### DISCUSSION TITLE
Change values with Home Assistant?

### DISCUSSION THREAD

**Discussion by sebska** [john30/ebusd#1159] [General]
Hi,

so i can see a lot of values in my Home Assistant now. But how can i change them (if they are, like the “Hc1HeatCurve”)?
An how must i configure it, that the values are displayed not with the “name” (Hc1HeatCurve) in the csv but with the “[comment]” (Heizkurve Heizkreis 1) (or simply change the "name" to "Heizkurve Kreis 1" in the csv?)?

Sebastian

![Values](https://github.com/john30/ebusd/assets/33956854/f3dfa11f-c0c1-4118-b303-d55ab464fdf0)

**Comment by Koky05** [comment]
In `mqtt-hassio.cfg` is defined filter for only read values you need to change it and the way depends on how you run ebusd.

**Comment by sebska** [comment]
ok, found it within mqtt-hassio.cfg
